scripts/o3de/o3de/manifest.py

```diff
@@ -404,6 +405,20 @@ def get_gem_templates(gem_path: pathlib.Path) -> list:
                         gem_object['templates'])) if 'templates' in gem_object else []
     return []
 
+def get_engines_json_data_by_path():
+    # use an OrderedDict to preserve the order found in the o3de manifest
+    engines_json_data = OrderedDict()
```

Comment:
Python dictionaries maintain order as of 3.7, so the OrderedDict collection is no longer needed.
https://mail.python.org/pipermail/python-dev/2017-December/151283.html

Reply:
oh cool! Not sure how I missed this.